src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoTunnel.kt

```diff
@@ -44,7 +44,15 @@ class AutoTunnel : Module() {
     }
 
     private fun sendTunnel() {
-        val current = arrayOf("tunnel", height.value.toString(), width.value.toString(), "1000000")
+        // fixed *censored* customsize tunnel if settings are default --pNoName
+        // im sorry for *censored* *censored* *censored* code im new to java i came from c++
```

Comment:
the comments don't need to be here, just add an `Updated by name on 25/05/20` in the @author section

Reply:
> the comments don't need to be here, just add an `Updated by name on 25/05/20` in the @author section

ok